src/content/docs/logs/log-management/get-started/get-started-log-management.mdx

```diff
@@ -23,43 +23,50 @@ As applications move towards the cloud, microservices architecture is becoming m
   id="3j7spmzlhc"
 />
 
-Our log management solution provides deeper visibility into application and infrastructure performance data (events and errors) to reduce mean-time-to-resolve (MTTR) and quickly troubleshoot production incidents. It does this by providing super-fast searching capabilities, alerts, and co-location of application, infrastructure, and log data, while visualizing everything from a single place.
+Our log management solution provides deeper visibility into application and infrastructure performance data (events and errors) to reduce mean-time-to-resolve (MTTR) and quickly troubleshoot production incidents. With everything integrated into New Relic One, you can use fast search capabilities, monitor alerts, review correlated data from your applications, infrastructure, and logs, and visualize everything from a single place.
```

Comment:
"monitor" is such a tricky word. Maybe something more like "keep an eye on alerts notifications"?

("alerts" is also a tricky word, doing triple duty as a product name, a collection of processes, and also as a colloquial stand-in for a notification)